client/src/Asset.tsx

```diff
@@ -156,46 +200,38 @@ function Asset(): JSX.Element {
                 <div className={classes.leftPanel}>
                     <div className={classes.imgs}>
                         <img src={bigImg} alt={asset.title} className={classes.bigImg} />
-                        <div className={classes.miniImgs}>
-                            {asset.imgUrls.map((imgUrl, ind) => ind !== selectedImgInd ? (
-                                <img
-                                    key={imgUrl}
-                                    src={imgUrl}
-                                    alt={asset.title}
-                                    className={classes.miniImg}
-                                    onClick={() => setSelectedImgInd(ind)}
-                                />
-                            ) : null)}
+                        <div  id="miniImgs" className={classes.miniImgs}>
+                            {showMiniImgs}
                         </div>
                     </div>
                     <p style={{ textAlign: 'left', padding: '20px 0' }}>
                         {asset.description}
                     </p>
                 </div>
                 <div className={classes.rightPanel}>
-                    <Typography variant="h3">
+                    <Typography className={classes.heading} variant="h3">
                         {asset.title}
                     </Typography>
                     <Typography className={classes.subText} variant="subtitle1">
                         {asset.categories.join(', ')}
                     </Typography>
-                    <Typography className={classes.subText} variant="subtitle1">
-                        Posted By {asset.postedBy}
-                    </Typography>
+                    {asset.organization ? "": "Posted By:"}{asset.postedBy.firstName}
```

Comment:
why remove the `Typography` here?
would be good to keep it in even if you change the styling or variant